src/pydap/model.py

```diff
@@ -287,18 +288,15 @@ class StructureType(DapType, Mapping):
     """A dict-like object holding other variables."""
 
     def __init__(self, name, attributes=None, **kwargs):
-        DapType.__init__(self, name, attributes, **kwargs)
+        super(StructureType, self).__init__(name, attributes, **kwargs)
 
         # emulate a simple ordered dict
```

Comment:
just a note: if you're not supporting Python 2.6 (which you shouldn't be, it's ancient!), then consider switching this to actually just use an OrderedDict.